pkl-gradle/src/main/java/org/pkl/gradle/PklPlugin.java

```diff
@@ -171,6 +171,8 @@ private void configureJavaCodeGenTasks(NamedDomainObjectContainer<JavaCodeGenSpe
           createModulesTask(JavaCodeGenTask.class, spec)
               .configure(
                   task -> {
+                    task.setDescription(TaskConstants.GENERATE_JAVA_DESCRIPTION);
+                    task.setGroup(TaskConstants.TASK_GROUP_CODEGEN);
```

Comment:
In general, this is how I'm configuring `description` and `group` on these tasks.